rules-tests/Php74/Rector/Property/RestoreDefaultNullToNullableTypePropertyRector/Fixture/skip_readonly_phpdoc_class.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\Php74\Rector\Property\RestoreDefaultNullToNullableTypePropertyRector\Fixture;
+
+/**
+ * @immutable
+ */
```

Comment:
I never use Immutable attribute, but found in a project, is it possible to skip:

```
#[\JetBrains\PhpStorm\Immutable]
```

and is that correct to skip that as well?

Reply:
my motivation to skip the phpdoc attribute is related to it beeing supported by phpstan.
(the rector rule when applied on classes tagged with `@immutable` will result in phpstan errors)

the jetbrains annotation is not supported by phpstan AFAIK, therefore not part of this PR